src/sentry/static/sentry/app/views/performance/transactionSummary/content.tsx

```diff
@@ -92,6 +97,15 @@ class SummaryContent extends React.Component<Props, State> {
     const totalCount = totalValues.count;
     const slowDuration = totalValues?.p95;
 
+    // NOTE: This is not a robust check for whether or not a transaction is a front end
+    // transaction, however it will suffice for now.
+    const hasWebVitals = VITAL_GROUPS.some(group =>
+      group.vitals.some(vital => {
+        const alias = getAggregateAlias(`percentile(${vital}, ${VITAL_PERCENTILE})`);
+        return totalValues[alias] !== null && !isNaN(totalValues[alias]);
```

Comment:
`Number.isFinite()` might be preferred here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite

`isNaN()` will coerce the value to a `Number`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#Description